detector_nodes:
# Keep in mind to tab-align the values!!!!
# The name of the detector MUST be the same as the field inside the DetectionArray message!
  - 
    name:           "BodyDetector"
    weight:         1.7
    min_detections: 1
    min_frequency:  2
    topic:          "/people_detections/body_detections"
  - 
    name:           "Face"
    min_detections: 1
    weight:         5
    min_frequency:  2
    topic:          "/people_detections/face_detections"
  - 
    name:           "laser"
    min_detections: 1
    weight:         5
    min_frequency:  2
    topic:          "/people_detections/laser_detections"
